src/tokens.json

```diff
@@ -347,6 +347,98 @@
       "type": "font",
       "comments": []
     },
+    "actionElementInnerText": {
+      "category": "typography",
+      "version": "1.0",
+      "value": {
+        "fontFamily": [
+          "Source Sans Pro",
+          "Helvetica Neue",
+          "Arial",
+          "sans-serif"
+        ],
+        "fontSize": {
+          "value": 16,
+          "unit": "px"
+        },
+        "lineHeight": {
+          "value": 1.5,
+          "unit": null
+        },
+        "fontWeight": 400
+      },
+      "type": "font",
+      "comments": []
+    },
+    "actionElementInnerTextSmallScreen": {
```

Comment:
Do we need the small screen variants as they appear to be identical with the regular counter part? Or is this a convention that is needed as other text tokens also have this?

Reply:
I think it is not generic enough if we don't define them as it can be fixed to be something different and then all implementations are missing usage.